app/presenters/publishing_api/world_location_news_article_presenter.rb

```diff
@@ -60,6 +60,16 @@ def image_available?
       item.images.any?
     end
 
+    def prepend_slash(path)
+      path.start_with?("/") ? path : path.prepend("/")
+    end
+
+    def image_path
+      image_path = presented_world_location_news_article.lead_image_path
+
+      Whitehall.public_asset_host + prepend_slash(image_path)
```

Comment:
You could also use `File.join(Whitehall.public_asset_host, image_path)` which would do the slashing for you. Not as 'correct' as the `URI` approach but you wouldn't need to handle errors.